tests/utils/test_sampling_utils.py

```diff
@@ -28,3 +30,42 @@ def test_sample_points_on_plane() -> None:
     np.testing.assert_almost_equal(pts_residuals, np.zeros((num_points, 1)))
 
     assert pts.shape == (10, 3)
+
+
+def test_sample_random_directions() -> None:
+    """Check that the sampled directions are uniformly distributed on the sphere, using their mean and std dev."""
+    num_samples = 1000
+
+    direction_list = sampling_utils.sample_random_directions(num_samples)
+    assert len(direction_list) == num_samples
+
+    directions = np.array([direction.point3() for direction in direction_list])
+    mean_direction = np.mean(directions, axis=0)
+    std_direction = np.std(directions, axis=0)
+    std_direction_relative = std_direction / std_direction[0]
```

Comment:
would it be helpful to add an inline comment explaining why we need to divide by standard dev here?

Reply:
magnitude is irrelevant here, added a comment.